### ISSUE TITLE
Auto refresh/reload table viewer

### ISSUE THREAD

**Issue by acarril** [REditorSupport/vscode-R#1014]
**Is your feature request related to a problem? Please describe.**
Table viewer does not reload automatically when the dataframe or matrix being shown changes. It is somewhat cumbersome to have to close that view and re-open it in order to reflect changes. I apologize if this has already been discussed, but I couldn't find a related issue.

**Describe the solution you'd like**
Ideally, the view of the table will refresh/reload automatically when the dataframe or matrix being displayed is modified.

**Describe alternatives you've considered**
A somewhat "inelegant" alternative would be to just implement a keybinding or command that closes the view of an object `foo` and then immediately calls `View(foo)` again.

**Additional context**
Below is a recording of what I mean, hoping it helps to make my point more clear. The simple example code is
```R
library(dplyr)
library(magrittr)

# Open table viewer for `df`
df <- starwars
View(df)

# Update `df`; problem is that changes are not reflected in viewer
df %<>%
  filter(height < 100)
```

https://user-images.githubusercontent.com/9773292/155612306-407ac62f-29ef-40a4-abf3-40f894b79b5c.mov

**Comment by albertosantini** [comment]
Many times I was about to write this feature request.

I didn't it because, in my context, I found useful to compare the previous values with the new ones.
Anyway this preference (refreshing or not refreshing) may be resolved with a setting or with a UI artifact making a call to action.

Curious to see extension team's replies.

**Comment by eliasyishak** [comment]
This would be a really nice feature to have, even if it's a flag we can turn on to auto refresh or keep stale to compare as @albertosantini mentioned

**Comment by jNorton519** [comment]
I just created an account to voice my support for this feature. To expand a little further, combining the power to pop out to a new window with auto refresh would be optimal for working with large dataframes. Thanks

**Comment by marveloushaotian** [comment]
Just wanted to check in and see if there's been any progress on implementing an auto-refresh for the table or list? Thanks for taking the time to work on this issue.

**Comment by v-vieira** [comment]
Would be amazing have this feature. So far this is one of the main reasons to keep using RStudio.

**Comment by ItsMeVictor** [comment]
Agreed that it wold be very nice to have!

**Comment by GB-IHE** [comment]
Agreed, I miss that.

**Comment by pabloreina97** [comment]
Agreed!!

**Comment by ajtwood** [comment]
Agreed!

**Comment by handsaway** [comment]
Would save so much time, agreed!

**Comment by scheerchristian** [comment]
Agree very much!

**Comment by GB-IHE** [comment]
I also want to know

**Comment by panarnur** [comment]
agree, this is extremely important to implement. is there any workaround anyone can suggest in the mean time?

**Comment by jensenl5** [comment]
I'd be thrilled if I could use Data Wrangler!
